sklearn/feature_selection/from_model.py

```diff
@@ -169,6 +169,12 @@ def fit(self, X, y=None, **fit_params):
         self.estimator_.fit(X, y, **fit_params)
         return self
 
+    @property
+    def threshold_(self):
+        scores = _get_feature_importances(self.estimator_, self.norm_order)
```

Comment:
I agree, the cost of evaluating the column-wise norm and the mean/median shouldn't be so significant. Can't think of use cases where this would be called in a tight loop.